src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -65,6 +65,61 @@ class LogicValueActionSerializer(serializers.Serializer):
     )
 
 
+class SynchronizeChildrenVariableMappingSerializer(serializers.Serializer):
+    property = serializers.CharField(
+        required=True, label=_("Property that needs mapping.")
+    )
+    component_key = FormioVariableKeyField(
+        required=True,
+        label=_("Key of the editgrid's child component."),
+    )
+
+
+class SynchronizeChildrenActionConfigSerializer(serializers.Serializer):
+    source_variable = FormioVariableKeyField(
+        required=True,
+        allow_blank=False,
+        label=_("Key of the form variable that will be used as the source."),
+    )
+    destination_variable = FormioVariableKeyField(
+        required=True,
+        allow_blank=False,
+        label=_("Key of the form variable that will be used as the destination."),
+    )
+    children_mappings = SynchronizeChildrenVariableMappingSerializer(
+        many=True,
+        label=_("Synchronize children mapping"),
+    )
```

Comment:
I think `data_mappings` is a better name and in fact, this entire serializer can probably be a bit more generic since I can imagine future situations where you want to map/synchronize from variable A to B with some sub-specification applied.